.github/workflows/schedule.yml

```diff
@@ -0,0 +1,64 @@
+name: 'E2E Testing Schedule'
+
+on:
+  schedule:
+    - cron: '0 3,7 * * *' # run at 8AM and 12PM (GTM+7)
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger Code Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js 18.15.0
+        uses: actions/setup-node@v1
+        with:
+          node-version: 18.15.0
+          registry-url: 'https://npm.pkg.github.com'
+          scope: '@kybernetwork'
+
+      - name: Install dependences
+        uses: bahmutov/npm-install@HEAD
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.GH_PAT }} ### authen npm
+
+      - name: Install linux deps
+        run: |
+            sudo apt-get install --no-install-recommends -y \
+            fluxbox \
+            xvfb
+  
+      - name: Yarn Build
+        env:
+          CI: false
+          VITE_TAG: ${{ needs.prepare.outputs.image_tag }}
+          CURRENT_BRANCH: ${{ needs.prepare.outputs.current_branch }}
+          NODE_OPTIONS: '--max_old_space_size=4096'
+        run: yarn build-adpr
```

Comment:
I think we should run regression on production env. 
yarn build-prod

Reply:
hmmm. cypress run on kyberswap.com, why do we need build step here?

Reply:
I've updated, please take a look at it. Thanks.